package.nls.es.json

```diff
@@ -0,0 +1,144 @@
+    "Override eclcc auto detection": "Anular la detección automática de eclcc",
+    "Password": "Contraseña",
+    "Pin": "Pin",
+    "Pin launch configuration": "Configuración de lanzamiento de pin",
+    "Pin launch configuration to current document": "Fijar la configuración de lanzamiento al documento actual",
+    "Pinned launch configurations": "Configuraciones de lanzamiento ancladas",
+    "Preview": "Presentación preliminar",
+    "Refresh": "Actualizar",
+    "Refresh Tree": "Actualizar árbol",
+    "Reject unauthorized calls e.g. SSL certificate errors": "Rechazar llamadas no autorizadas, por.ej. Errores de certificado SSL",
+    "Reveal Generated ECL in File Explorer": "Revelar ECL generado en el Explorador de archivos",
+    "Reveals Generated ECL in File Explorer": "Revela ECL generada en el Explorador de archivos",
+    "Run 'eclcc -syntax' on load": "Ejecute 'eclcc -syntax' en la carga",
+    "Run 'eclcc -syntax' on save": "Ejecute 'eclcc -syntax' al guardar",
+    "Save + check syntax of current file": "Guardar y verificar la sintaxis del archivo actual",
+    "Save All + check syntax of all files": "Guardar todo y comprobar la sintaxis de todos los archivos",
+    "Select Client Tools Version": "Seleccione la versión de las herramientas cliente",
+    "Select Client Tools Version from available options": "Seleccione la versión de herramientas cliente de las opciones disponibles",
+    "Select KEL version": "Seleccione versión de Kel",
+    "Select eclcc version": "Seleccione versión de eclcc",
+    "Server IP/Domain address": "Dirección IP / dominio del servidor",
+    "Server protocol (http or https)": "Protocolo de servidor (http o https)",
+    "Seven Weeks Ago": "Hace siete semenas",
+    "Showing \"my\" workunits": "Mostrando mis uinidades de trabajo",
+    "Showing all workunits": "Mostrando todos los unidades de trabajo",
+    "Six Weeks Ago": "Hace seis semanas",
+    "Submit": "Enviar",
+    "Submit ECL": "Enviar ECL",
+    "Submit ECL to HPCC Platform": "Enviar ECL a HPCC Systems®",
+    "Submits ECL to public demo servers, hthor": "Envía ECL a servidores públicos de demostración, hthor",
+    "Submitted workunit": "Unidades de trabajo sormitidos",
+    "Submitting ECL via the Run/Debug page is being deprecated.  Please use the new Submit + Compile buttons at the top of the ECL Editor": "El envío de ECL a través de la página Ejecutar / Depurar está obsoleto. Utilice los nuevos botones Enviar + Compilar en la parte superior del Editor ECL",
+    "Submitting workunit": "Enviar unidad de trabajo",
+    "Switch HPCC Platform": "Cambiar HPCC Platform",
+    "Switch Target Cluster": "Cambiar sistemas de destino",
+    "Switch to different HPCC Platform instance": "Cambiar a una instancia de plataforma HPCC diferente",
+    "Syntax Check": "Comprobación de sintaxis",
+    "Syntax Check all files": "Comprobar sintaxis a todos los archivos",
+    "Syntax Clear": "Sintaxis clara",
```

Comment:
not sure what we're trying to convey here.... but "sintaxis clara" means "clear syntax".
Is this a command to clear all syntax check output? In that case, we could say "Borrar sintaxis" which would translate back to "erase syntax"